docs/examples/move/nft_marketplace/sources/item_for_market.move

```diff
@@ -0,0 +1,58 @@
+module nft_marketplace::market_items {
```

Comment:
Would be nice to make it more specific, would it be nice to call the module (file and OTW) like.
Also add a small coment that describes the module.

Reply:
Sorry I missed the example in the copy and paste! I meant something like `clothing_store`

Reply:
Done